File: exercises-Java/soma_vetor.java
import java.util.Locale;
import java.util.Scanner;

public class soma_vetor {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		int N;
		double media, soma;
		System.out.println("Quantos numeros serao digitados? ");
		N = sc.nextInt();
		double[] vet = new double [N]; // vetor começa em 0.
		for(int i = 0; i < N; i++) {
			System.out.print("Digite um nuemro: ");
			vet[i] = sc.nextDouble();
		}
		System.out.println();
		System.out.println("Valores = ");
		for(int i = 0; i < N; i++) {
			System.out.print(" " + String.format("%.1f", vet[i]));
		}
		System.out.println();
		soma = 0;
		for(int i = 0; i < N; i++) {
			soma = soma + vet[i];
		}
		System.out.println("Soma = " + String.format("%.2f", soma));
		media = soma / N;
		System.out.println("Media = " + String.format("%.2f", media));
		sc.close();
	}

}

File: exercises-Python/soma_vetor.py
N = int(input("Quantos números serão digitados? "))
vet: [float] = [0 for x in range (N)]
for i in range(0, N):
    vet[i] = float(input("Digite um número: "))
print()
print("Valores = ", end="")
for i in range(0, N):
    print(f"{vet[i]:.1f} ", end="")
print()
soma = 0
for i in range(0, N):
    soma = soma + vet[i]
media = soma / N
print(f"Soma = {soma:.1f}")
print(f"Média = {media:.1f}")
